microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -96,33 +96,33 @@ Once you have completed the setup, return to this page and continue with the nex
-## Step 2 - Set up Unity project for HoloLens 2 development
+## Step 2 - Set up Unity for HoloLens 2 development
 
-Before you can start developing a HoloLens 2 app with WebView2, you'll need to set up your Unity project for HoloLens 2 development. To prepare your project for mixed reality development, follow the steps in [Unity development for HoloLens 2](/windows/mixed-reality/develop/unity/unity-development-overview?tabs=arr,D365,hl2) in the Mixed Reality documentation. For a step-by-step walkthrough of configuring a Unity project, see [Set up a new OpenXR project with MRTK](/windows/mixed-reality/develop/unity/new-openxr-project-with-mrtk).
+Before you can start developing a HoloLens 2 app with WebView2, you will need to install Unity.  Follow the steps in [Choosing a Unity version and XR plugin](/windows/mixed-reality/develop/unity/choosing-unity-version) to complete installation of the toolset needed to build Mixed Reality applications.
 
 
 <!-- ====================================================================== -->
-## Step 3 - Install Mixed Reality Feature Tool and Mixed Reality Toolkit (MRTK)
+## Step 3 - Configure the Unity project for Mixed Reality development
 
-Once you have your Unity project ready for HoloLens 2 development, you'll need to configure the Mixed Reality Toolkit (MRTK) in your project.
+Once you have your tools installed, you're ready to create and configure your project.   The easiest way to get started with HoloLens 2 development is to use the the Mixed Reality Toolkit (MRTK) in your project.  
 
-1. First, [Download the Mixed Reality Feature Tool](https://www.microsoft.com/download/details.aspx?id=102778) from the Microsoft Download Center.
+The Mixed Reality Feature Tool simplifies the process of discovering, installing, and managing Mixed Reality features, including the Mixed Reality Toolkit, and packages for your Unity project. For more guidance on installing the Mixed Reality Feature Tool, see [Welcome to the Mixed Reality Feature Tool](/windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool).
 
-   The Mixed Reality Feature Tool simplifies the process of discovering, installing, and managing Mixed Reality features and packages for your Unity project. For more guidance on installing the Mixed Reality Feature Tool, see [Welcome to the Mixed Reality Feature Tool](/windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool).
+1. First, [Download the Mixed Reality Feature Tool](https://www.microsoft.com/download/details.aspx?id=102778) from the Microsoft Download Center.   
 
 1. Run the downloaded Mixed Reality Feature Tool executable, and follow the prompts to install the Mixed Reality Toolkit (MRTK) and any additional required packages for HoloLens 2 development.
 
    The Mixed Reality Feature Tool allows you to select and install the appropriate versions of the Mixed Reality Toolkit (MRTK), along with other Mixed Reality features, directly into your Unity project.
```

Comment:
```suggestion
   The Mixed Reality Feature Tool allows you to select and install the appropriate versions of the Mixed Reality Toolkit (MRTK), along with other mixed reality features, directly into your Unity project.
```
if the meaning here is more "general industry tech" instead of "specifically Microsoft products"